clients/dotnet-core-rest/src/Tika.RestClient/Factories/RestClientFactory.cs

```diff
@@ -6,19 +6,28 @@ namespace Tika.RestClient.Factories
 {
     public static class RestClientFactory
     {
-        public static IRestClient Create(HttpClient httpClient)
+        public static IRestClient Create(HttpClient httpClient, ClientOptions options, string cluster = "")
```

Comment:
Default values in .net core should be assigned using default keyword so we dont have to change the right hand side of the assignment if we change the type on the left hand side!